IPython/html/static/notebook/less/notebook.less

```diff
@@ -5,6 +5,32 @@
         padding-right: 0px;
     }
 }
+/* Sticky header effect. Includes media queries to keep 
+   margin between #site and #header when resizing. 
+ */
+
+.header_notebook {
+    position: fixed;
+    width: 100%;
+}
+
+.site_notebook {
+    margin-top: 9em;
+}
+
+@media screen and (-webkit-min-device-pixel-ratio:0) and (max-width: 636px) {
+      .site_notebook {
+          margin-top: 11em;
+        }
+}
+
+@media (max-width: 633px) {
```

Comment:
No, that is used to have the notebook below the header. Since a fixed header removes it from the flow of the document, you always need to add a `margin-top` to get a correct layout. The number comes (approximately) from the height of the header. The media queries can be removed but they are supposed to fix the issue I described in the first commit (when resizing the window, the header increases its size to fit its elements and the margin between header and page is reduced.) We could dynamically calculate the height but I really want to avoid that solution if it is possible.